src/SelfGravity/Poi_UserWorkBeforePoisson_GREP.cpp

```diff
@@ -31,11 +31,10 @@ double GREP_Prof_Center   [3];
 
 extern real *h_ExtPotGREP;
 
-// temporary switch for test different schemes for temporal interpolation:
-//   True : apply temporal interpolation in Aux_ComputeProfile()
-//   False: apply temporal interpolation when combining the stored profiles
-static bool Do_TEMPINT_in_ComputeProfile = true;
-
+// switch for when the temporal interpolation is applied
+//   true : in Aux_ComputeProfile()
+//   false: when combining the stored profiles
+static bool GREP_OPT_TEMPINT = false;
```

Comment:
Just a naming suggestion. Is it better to use `TIMEINTERP` instead of `TEMPINT`. `TEMPINT` looks ike inital temperature.

Reply:
I will update it later.